envs/scib-pipeline.yml

```diff
@@ -16,21 +17,21 @@ dependencies:
   - libgcc-ng
   - numba<=0.46  # for mnnpy
   - anndata2ri
-  - r-base
   - r-essentials
   - r-devtools
   - r-stringi
   - bioconductor-scater
   - bioconductor-scran
   # Methods
+#  - mnnpy=0.1.9.5
   - scvi=0.6.7
   - scanorama=1.7.0
   - bbknn=1.3.9
   - r-seurat=3.1.1
   - pip:
     - scib==1.0.0
     - trvaep==0.1.0
-    - mnnpy==0.1.9.5
+    - git+https://github.com/pinin4fjords/mnnpy.git
```

Comment:
why the mnnpy change for the old env? Is this due to dependency clashes w/ an old MNN version that hasn't been updated?